packages/ui/src/components/va-skip-link/VaSkipLink.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <a
```

Comment:
Can we also apply it to vuestic docs? Would be interesting to test if this works properly (doesn't trigger reload etc).